ui/src/utils/realtime/implementation.ts

```diff
@@ -0,0 +1,121 @@
+import { neverEver } from '@/utils/never-ever';
+import { createRealtimeApiConnection } from '@/utils/realtime/realtime.signalr';
+import type { HubConnection } from '@microsoft/signalr';
+import { HubConnectionState } from '@microsoft/signalr';
+import type {
+	HubStatusMesage,
+	MessageFromServer,
+	MessageFromApp,
+} from './messages';
+/// <reference types="vite/client" />
```

Comment:
Not familiar with this, but i'm guessing it's useful for something Vite related?  I'm just leaving this comment here to draw attention in case it's not intentional

Reply:
It was not in the final form; should have been removed with the "Remove unnecessary PWA implementation" commit.